Simplified/Audiobooks/NYPLLastListenPositionSynchronizer.swift

```diff
@@ -0,0 +1,125 @@
+//
+//  NYPLLastListenPositionSynchronizer.swift
+//  Simplified
+//
+//  Created by Ernest Fan on 2022-11-10.
+//  Copyright © 2022 NYPL. All rights reserved.
+//
+#if FEATURE_AUDIOBOOKS
+import Foundation
+import NYPLAudiobookToolkit
+
+class NYPLLastListenPositionSynchronizer: NYPLLastListenPositionSynchronizing {
+  private let book: NYPLBook
+  private let bookRegistryProvider: NYPLBookRegistryProvider
+  private let annotationsSynchronizer: NYPLLastReadPositionSupportAPI
+  private let deviceID: String?
```

Comment:
the NYPLLastListenPositionSynchronizer is only created when we open the audiobook, correct?

Reply:
Right, an instance of the synchronizer is being created when we present the audiobook, and released when the audiobook is being closed.